docs/source/sysadmin_guide/configuring_permissions.rst

```diff
@@ -84,106 +121,114 @@ transactions. Each role equals a policy name that corresponds to a policy
 stored in state. The policy contains a list of public keys that correspond
 to the identity signing key of the transactors.
 
-To configure on-chain roles, the signer of identity transactions needs to have
-their public key set in the Setting "sawtooth.identity.allowed_keys". Only
-those transactors whose public keys are in that setting are allowed to update
-roles and policies.
+Before you can configure on-chain roles, your public key must be set in the
+``sawtooth.identity.allowed_keys`` setting. Only those transactors whose
+public keys are in that setting are allowed to update roles and policies.
 
-.. code-block:: console
+1. Make sure that the Identity transaction processor and the REST API are
+   running, as described in :doc:`systemd`.
 
-  $ sawset proposal create sawtooth.identity.allowed_keys=02b2be336a6ada8f96881cd55fd848c10386d99d0a05e1778d2fc1c60c2783c2f4
+#. Use the following command to add a signer's public key to this setting:
 
-Once your signer key is stored in the setting, the ``identity-tp`` command can be used
-to set and update roles and policies. Make sure that the Identity transaction
-processor and the REST API are running.
+   .. code-block:: console
 
-.. literalinclude:: ../cli/output/sawtooth_identity_policy_create_usage.out
-  :language: console
+     $ sawset proposal create sawtooth.identity.allowed_keys=02b2be336a6ada8f96881cd55fd848c10386d99d0a05e1778d2fc1c60c2783c2f4
```

Comment:
maybe we should mention exactly what key is needed here. In some of my scripts I do `sawtooth.identity.allowed_keys=$(cat /home/sawtooth/.sawtooth/keys/sawtooth.pub)`